code/game/objects/structures/plasticflaps.dm

```diff
@@ -8,7 +8,6 @@
 	layer = MOB_LAYER
 	plane = MOB_PLANE
 	explosion_resistance = 5
-	var/can_pass_lying = 1
```

Comment:
Sorry for the delay, but feedback from devs is that we're fine with the removal of the recipe, but want it to continue to block mobs from passing through.

Reply:
Changes applied